pyaedt/misc/Jupyter.py_build

```diff
@@ -0,0 +1,48 @@
+"""
+* * * This script is meant to run in IronPython within the Ansys Electronics Desktop * * *
+
+It looks for a reference to a python interpreter in python_interpreter.bat
+
+This script executes the "hidden" CPython script console_setup (hidden because the .py was omitted)
+This hidden script sets up the hfss variable based on the command line arguments and kicks off the interactive python
+shell.
```

Comment:
```suggestion
* * * This script is meant to run in IronPython within Ansys Electronics Desktop. * * *

It looks for a reference to a Python interpreter in the ``python_interpreter.bat`` file.

This script executes the "hidden" CPython ``console_setup`` script (hidden because the PY file was omitted).
This hidden script sets up the HFSS variable based on the command line arguments and kicks off the interactive Python shell.
```